requirements.txt

```diff
@@ -3,3 +3,4 @@ anytree>=2.8.0
 stringcase>=1.2.0
 deprecation>=2.1.0
 pytest>=2.7.2
+rdflib>=6.0.0
```

Comment:
please add (and use `Pipfile`) instead, it's just a matter of getting used and remember that, but much more handy than `requirements.txt` as yo don't mess with your system.

You can keep this one, but please consider adding to `Pipfile` as well.

Hint: if you `export PIPENV_VENV_IN_PROJECT=1` in your shell, then it will create a `.venv` foder and you keep all that stuff per project :-) Then when you `pipenv shell` you load that environment, when you exit that, it's all clean. Also `git clean -xfd` will remove everything, including such dependencies.

Reply:
Thanks for the pointer. Will fix it!